molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/{{cookiecutter.app_name}}/templates/core/tags/bannerpages.html

```diff
@@ -5,12 +5,14 @@
 {% for bannerpage in bannerpages %}
 <div class="header">
   {% if bannerpage.banner %}
-    {% if bannerpage.banner_link_page %}
-    <a href="{% pageurl bannerpage.banner_link_page %}">{% image bannerpage.banner width-320 %}</a>
+	    {% if bannerpage.banner_link_page %}
+	    <a href="{% pageurl bannerpage.banner_link_page %}">{% image bannerpage.banner width-320 %}</a>
+	    {% else %}
+	    {% image bannerpage.banner width-320 %}
+	    {% endif %}
     {% else %}
-    {% image bannerpage.banner width-320 %}
-    {% endif %}
-  {% endif %}
+    {% external_link %}
```

Comment:
@Ntu2koTrevor this should be `{% bannerpage.external_link %}` also please implement it in the same way `bannerpage.banner_link_page` has been implemented. Above line 10 you need to add another else statement checking if the banner has an external_link and then adding the page URL for the external link